src/endpoint/s3/ops/s3_get_object.js

```diff
@@ -15,14 +15,21 @@ async function get_object(req, res) {
     const noobaa_trigger_agent = agent_header && agent_header.includes('exec-env/NOOBAA_FUNCTION');
     const encryption = s3_utils.parse_encryption(req);
 
-    const object_md = await req.object_sdk.read_object_md({
+    let get_from_cache = false;
+    if (req.query.get_from_cache === 'true') {
+        get_from_cache = true;
+    }
+    let md_params = {
```

Comment:
```suggestion
    const params = {
```
You should still use `const` and not `let` because const only applies to the reference in javascript and not to the object itself. For objects to be immutable you use `Object.freeze(obj)` or even `js_utils.deep_freeze(obj)`.

Also the name `params` would be more compatible with the name inside the call.

Reply:
ok